tests/unit/Newsletter/RendererTest.php

```diff
@@ -209,7 +227,7 @@ function testItRendersHeader() {
   function testItRendersImage() {
     $newsletter = $this->newsletter['body'];
     $template = $newsletter['content']['blocks'][0]['blocks'][0]['blocks'][1];
-    $DOM = $this->DOM_parser->parseStr(Image::render($template, $columnCount = 1));
+    $DOM = $this->DOM_parser->parseStr(Image::render($template, 660));
```

Comment:
I think it would be nicer to use `const COLUMN_BASE_WIDTH`. One would still understand what it is and the value would be defined in one place.